component-library/src/components/LayerListControls/reorder-layers-array.js

```diff
@@ -0,0 +1,50 @@
+import moveInArray from "../../utils/move-in-array";
+
+const isLayerGroup = item => Boolean(item.children)
```

Comment:
This is value is never used in this file

Reply:
thanks